Mathlib/RepresentationTheory/GroupCohomology/Resolution.lean

```diff
@@ -357,21 +349,21 @@ theorem diagonalHomEquiv_symm_apply (f : (Fin n → G) → A) (x : Fin (n + 1) 
     Category.comp_id, Action.comp_hom, MonoidalClosed.linearHomEquivComm_symm_hom]
   -- Porting note: This is a sure sign that coercions for morphisms in `ModuleCat`
   -- are still not set up properly.
-  -- This used to be `rw`, but we need `erw` after leanprover/lean4#2644
   rw [ModuleCat.coe_comp]
   simp only [ModuleCat.coe_comp, Function.comp_apply]
   -- This used to be `rw`, but we need `erw` after leanprover/lean4#2644
   erw [diagonalSucc_hom_single]
-  erw [TensorProduct.uncurry_apply, Finsupp.lift_apply, Finsupp.sum_single_index]
-  · simp only [one_smul]
-    erw [Representation.linHom_apply]
-    simp only [LinearMap.comp_apply, MonoidHom.one_apply, LinearMap.one_apply]
-    erw [Finsupp.llift_apply]
-    rw [Finsupp.lift_apply]
-    erw [Finsupp.sum_single_index]
-    · rw [one_smul]
+  conv_lhs => -- Wrapped in `conv_lhs` because testing for `rw`-ability is too slow.
```

Comment:
Is this comment meant to land? (I wonder if it was added when the linter was still present.)

Reply:
```suggestion
  -- The prototype linter that checks if `erw` could be replaced with `rw` would time out
  -- if it replaces the next `erw`s with `rw`s. So we focus down on the relevant part.
  conv_lhs =>
```